src/PetriEngine/Colored/Reduction/ColoredReducer.cpp

```diff
@@ -89,6 +89,7 @@ namespace PetriEngine::Colored::Reduction {
 
     void ColoredReducer::skipPlace(uint32_t pid) {
         Place &place = _builder._places[pid];
+        printf("-> Rem P: %s\n", place.name.c_str());
```

Comment:
Not necessary to have debug stuff on main

Reply:
Fixed in 002815c.